mmv1/third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -402,10 +405,10 @@ in Terraform state, a `terraform destroy` or `terraform apply` that would delete
 * `dataset_id`: (Required) The ID of the dataset containing this table.
 
 * `table_id`: (Required) The ID of the table. The ID must contain only
-	letters (a-z, A-Z), numbers (0-9), or underscores (_). The maximum
-	length is 1,024 characters. Certain operations allow suffixing of
-	the table ID with a partition decorator, such as
-	sample_table$20190123.
+ letters (a-z, A-Z), numbers (0-9), or underscores (_). The maximum
```

Comment:
Same question here, does this render the way you intended? I would expect a second space to match the indentation used elsewhere.